scripts/build-circt.sh

```diff
@@ -30,12 +33,17 @@ while [[ "$#" -ge 1 ]] ; do
 	case "$1" in
 		--build-path)
 			shift
-			CIRCT_BUILD="${PWD}/$1"
+			CIRCT_BUILD=$(readlink -m "$1")
```

Comment:
Reading the manual for readlink it is for getting the path of symbolic links. But there shouldn't be any symbolic links in this case so I don't understand it's purpose.

Reply:
Your main argument for this script was convenience for the user. The script is supposed to be used by the user and we do not know what the user provides as input. The `$(readlink -m "$1")` takes a given absolute or relative path (with symlinks) and gives me back an absolute path (that also does not need to exist).